examples/siro_sandbox/server/client_message_manager.py

```diff
@@ -38,3 +38,10 @@ def add_highlight(self, pos: List[float], radius: float) -> None:
         self._message["highlights"].append(
             {"t": [pos[0], pos[1], pos[2]], "r": radius}
         )
+
+    def change_humanoid_position(self, pos: List[float]) -> None:
+        r"""
+        Change the position of the humanoid.
+        Used to synchronize the humanoid position in the client when changing scene.
+        """
+        self._message["humanoidPosition"] = [pos[0], pos[1], pos[2]]
```

Comment:
How about a better name like "forceAvatarBasePosition" (player? user?). "Humanoid" is overly specific, and I want "base" because you are specifying a location for the base (the feet). (We can imagine in the future, for multi-story houses, the y coord will be more significant.)